toolkits/global/packages/global-skip-link/scss/10-settings/_springernature.scss

```diff
@@ -11,3 +11,4 @@ $skip-link--font-size: 1rem;
 $skip-link--color: color('white');
 $skip-link--hover-color: color('white');
 $skip-link--background-color: color('universal-blue');
+$skip-link--z-index: 9999;
```

Comment:
an aside (so apologies for dropping it here) … but perhaps there's chance for a more consistent, scalable `z-index` naming strategy from Design Tokens to Sass variables. 

Something like's mentioned [in this article](https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/)

```
$zindex-dropdown:       1000;
$zindex-sticky:         1020;
$zindex-fixed:          1030;
$zindex-modal-backdrop: 1040;
$zindex-modal:          1050;
$zindex-popover:        1060;
$zindex-tooltip:        1070;
```

(note: there are _a lot_ of articles on z-index, naming, and Sass variables (I think I read them all when I curated the weekly Sass newsletter 😭)